### DISCUSSION TITLE
Harden Android device

### DISCUSSION THREAD

**Discussion by Bruzzler0** [DesktopECHO/Pi-hole-for-Android#54] [Q&A]
Can anyone provide me with tips on how to harden my rooted Android smartphone? I have installed the AFWall+ firewall, and the operating system is Lineage OS. I plan to use the smartphone as a DNS server (Pihole) on my network

**Comment by DesktopECHO** [comment]
If you're using a device from a reputable manufacturer and/or have LineageOS installed, I would say you're starting off in pretty good shape!
